stack-ghc844.yaml

```diff
@@ -0,0 +1,3 @@
+resolver: lts-12.15 # first lts-12.X which uses ghc844
```

Comment:
IIRC we had to use `ghc844` because `ghc843` was not provided by the pinned version of nixpkgs. Is this still relevant now we use the haskellNix overlay? I.e. could this be `lts-12.0` again now?

Reply:
Yes it is still relevant. Apparently `haskell.nix` picks up compilers from what is already defined in nixpkgs.

Reply:
> Apparently haskell.nix picks up compilers from what is already defined in nixpkgs.

I started `nix-shell` and right now it is compiling GHC, so I don't think GHC is being picked up from nixpkgs - otherwise it would just download the built version from Hydra, no?

Reply:
I do not have the answer. I only know that if `ghc843` is not present in nixpkgs it fails.